app/assets/javascripts/slug-generation.js

```diff
@@ -1,17 +1,48 @@
 $(function() {
+  var $form = $(".js-guide-form");
+
+  if ($form.length == 0) {
```

Comment:
How about just removing it entirely. It doesn't really add much in terms of performance, and the code below will not break without it: https://github.com/alphagov/service-manual-publisher/pull/253/commits/ac41101eaa6883f483878373f4437b0c7c8823c9